### PR TITLE
Clean up CI scripts and prune Docker images between builds

### PR DESCRIPTION
**Pull request by yurishkuro**
## Which problem is this PR solving?
- Current ci-release workflow is running out of disk space

## Description of the changes
- 

## How was this change tested?
- Cannot test until merged

### CODE REVIEW

File: scripts/build-all-in-one-image.sh [hunk 1]
```diff
@@ -2,7 +2,6 @@
 
 set -exu
 
-BRANCH=${BRANCH:?'missing BRANCH env var'}
```
Comment (yurishkuro) [inline scripts/build-all-in-one-image.sh:5]: not used in this script

File: scripts/hotrod-integration-test.sh [hunk 2]
```diff
@@ -26,5 +26,4 @@ if [[ $body != *"Rides On Demand"* ]]; then
 fi
 docker rm -f $CID
 
-BRANCH=${BRANCH:?'missing BRANCH env var'}
```
Comment (yurishkuro) [inline scripts/hotrod-integration-test.sh:29]: not used in this script

File: scripts/rebuild-ui.sh [hunk 3]
```diff
@@ -7,16 +7,16 @@ cd jaeger-ui
...
+if [[ "$last_tag" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]];  then
+    branch_hash=$(git rev-parse HEAD)
+    last_tag_hash=$(git rev-parse $last_tag)
 
-    if [[ "$BRANCH_HASH" == "$LAST_TAG_HASH" ]]; then
+    if [[ "$branch_hash" == "$last_tag_hash" ]]; then
         temp_file=$(mktemp)
         trap "rm -f ${temp_file}" EXIT
-        release_url="https://github.com/jaegertracing/jaeger-ui/releases/download/${LAST_TAG}/assets.tar.gz"
+        release_url="https://github.com/jaegertracing/jaeger-ui/releases/download/${last_tag}/assets.tar.gz"
```
Comment (yurishkuro) [inline scripts/rebuild-ui.sh:19]: just a refactoring to use lowercase names for variables

File: scripts/build-upload-a-docker-image.sh [hunk 4]
```diff
@@ -71,3 +71,6 @@ docker buildx build --output "${PUSHTAG}" \
 echo "Finished building${upload_flag} ${component_name} =============="
 
 df -h /
+docker buildx prune --all --verbose
+docker system prune
```
Comment (albertteoh) [inline scripts/build-upload-a-docker-image.sh:75]: We might need to add `--force` for each of these to avoid being prompted for confirmation.

**Pull request merged**